src/auxiliary/auxiliary.jl

```diff
@@ -163,28 +158,28 @@ whenever MPI is used.
 For example, use `solve(ode, alg; ode_default_options()...)`
 """
 function ode_default_options()
-  if mpi_isparallel()
-    return (; save_everystep = false, internalnorm = ode_norm, unstable_check = ode_unstable_check)
-  else
-    return (; save_everystep = false)
-  end
+    if mpi_isparallel()
+        return (; save_everystep = false, internalnorm = ode_norm,
+                unstable_check = ode_unstable_check)
```

Comment:
This still looks a bit unusual to me - but maybe we need to live with it, too

Reply:
You mean the alignment of `unstable_check` with the `;` instead of `save_everystep`? Yeah, I think we need to live with that.

I am not so sure about the whitespace around kwargs - I belivee it is very common in Julia to not have whitespace there...

Reply:
Yes, that's what I mean. Having whitespace around keyword arguments is fine with me